src/cogent3/app/io.py

```diff
@@ -102,7 +102,7 @@ def get_data_store(
     return open_data_store(base_path, suffix, limit, verbose)
 
 
-def _load_seqs(path, klass, parser, moltype):
+def _load_seqs(path, klass, parser, moltype):  # pragma: no cover
     abs_path = str(path)
     if not hasattr(path, "read"):
         # we use a DataStoreMember as it's read() handles zipped compression
```

Comment:
Does this function need deprecated warning?

Reply:
No. There is a modified version in `io_new.py`. I;ve left this one in place as it's behaviour is different due to working with the older data store classes.